post/forms.py

```diff
@@ -5,7 +5,4 @@ class PostForm(forms.ModelForm):
 
     class Meta:
         model = Post
-        fields = ['title','cover']
-        # fields['owner']= request.user.id
```

Comment:
[trivial]
:+1: for removing commented code, but try to remember python files should end in a newline